backend/src/main/java/eliasyazdani/capstone/cupwithme/backend/tournament/TournamentService.java

```diff
@@ -22,24 +22,33 @@ public List<Tournament> getAllTournamentsForAdmin(String admin) {
     }
 
     public Tournament addNewTournament(TournamentWithoutID tournamentWithoutID) {
-        List<Match> newMatches = new ArrayList<>();
```

Comment:
I wouldn't take a keyword like _new_ for naming it. An appropriate name is just _rounds_ for this list.

Reply:
I am creating new Tournament with new rounds and metches. Thanks for your suggestion.